docs/how-to-guides/develop-a-tool/use-file-path-as-tool-input.md

```diff
@@ -0,0 +1,98 @@
+# Use FilePath as tool input
```

Comment:
Better to introduce the What is FilePath type and its capability with a screenshot to let user know that it can pick up a file in UI.